### PR TITLE
Update poetry deps and gitignore and spellcheck mechanizm 

### PR DESCRIPTION
**Pull request by thunderbirdtr**
I would like to suggest that we should use "pyspelling" which helps me a lot on typo check and I already build a dictionary for this documentation we can use to improve spell check and helps to prevent typos as well.

### CODE REVIEW

**Comment by why-not-try-calmer** [comment]
> I would like to suggest that we should use "pyspelling" which helps me a lot on typo check and I already build a dictionary for this documentation we can use to improve spell check and helps to prevent typos as well.

Sounds great, can you briefly explain how this works? Perhaps in an upcoming meeting?

**Comment by thunderbirdtr** [comment]
> > I would like to suggest that we should use "pyspelling" which helps me a lot on typo check and I already build a dictionary for this documentation we can use to improve spell check and helps to prevent typos as well.
> 
> Sounds great, can you briefly explain how this works? Perhaps in an upcoming meeting?

Absolutely, that would be my pleasure to show and discuss with you guys and in a meeting. 

TLDR version : Basically we execute "pypspelling" CLI tools for checking spells errors and outputs are basically shows you "typo and unknown" words. Once add "unknown" words into dictionary such as "openSUSE" rest will be "typos" and we can fix in docs.

**Comment by why-not-try-calmer** [comment]
> > > I would like to suggest that we should use "pyspelling" which helps me a lot on typo check and I already build a dictionary for this documentation we can use to improve spell check and helps to prevent typos as well.
> > 
> > 
> > Sounds great, can you briefly explain how this works? Perhaps in an upcoming meeting?
> 
> Absolutely, that would be my pleasure to show and discuss with you guys and in a meeting.
> 
> TLDR version : Basically we execute "pypspelling" CLI tools for checking spells errors and outputs are basically shows you "typo and unknown" words. Once add "unknown" words into dictionary such as "openSUSE" rest will be "typos" and we can fix in docs.

Sounds good. And about "whielisting" these unknown words: you need to copy-paste them from an .md file into some config file used by this tool?
